.github/workflows/pr-integration-tests.yml

```diff
@@ -4,11 +4,11 @@ on:
   pull_request:
     branches:
       - master
-      - branch-*
+      - "release/v[0-9]+.[0-9]+"
```

Comment:
Can we use `v[0-9]+.[0-9]+.[0-9]+.*` so that we can support the RC release or some special releases like `v1.2.3-rc1`